test/execflags/configFile/configFileAmbiguousVar.cfg

```diff
@@ -0,0 +1 @@
+name="brad"
```

Comment:
I didn't realize this was a file format, I've always used .compopts.  Neat!

Reply:
Oohhh, never mind, you included it explicitly.  Gotcha

Reply:
I think it would be clearer to just put the config value in the .execopts file so people don't have to look in multiple files to figure out what is going on